File: 2015/java/src/test/java/nl/sbmf21/aoc15/days/Day8Test.java
package nl.sbmf21.aoc15.days;

import org.junit.jupiter.api.Test;

import static nl.sbmf21.aoc.testing.UtilKt.testDay;

public class Day8Test {

    @Test
    public void testInput() {
        testDay(Day8.class, 1342, 2074, false, null);
    }

    @Test
    public void testExample() {
        testDay(Day8.class, 12, 19, true, null);
    }
}

File: 2022/kotlin/src/test/kotlin/nl/sbmf21/aoc22/days/Day8Test.kt
package nl.sbmf21.aoc22.days

import nl.sbmf21.aoc.testing.testDay
import org.junit.jupiter.api.Test

class Day8Test {

    @Test
    fun testInput() = testDay(Day8::class.java, 1814, 330_786)

    @Test
    fun testExample() = testDay(Day8::class.java, 21, 8, true)
}